react/ComponentDocs.tsx

```diff
@@ -1,15 +1,13 @@
-import React, { Fragment, FunctionComponent } from 'react'
+import React, { FunctionComponent } from 'react'
```

Comment:
you can use `FC` instead of `FunctionComponent `

Reply:
Swapped `FunctionComponent` for `FC` in the other components aswell